lib/index.js

```diff
@@ -0,0 +1,38 @@
+const { Pool } = require('pg');
+
+class Chatbase {
+  constructor() {
+    this.pool = new Pool({
+      user: process.env.user || argValidation('user'),
```

Comment:
Why lowercase env vars?? I would always do uppercase, more canonical. Also, why not use ESNext default arguments if you want what are essentially optional arguments?

Reply:
Ops distraction error, my bad! I always follow the upper case convention, i'll switch to default argument!